crml/staking/src/lib.rs

```diff
@@ -726,6 +726,9 @@ decl_storage! {
 		/// Rewards for the current era. Using indices of current elected set.
 		CurrentEraPointsEarned get(fn current_era_reward): EraPoints;
 
+		/// Total transaction payment rewards for elected validators
+		CurrentEraTransactionRewards : BalanceOf<T>;
```

Comment:
[Suggestion] To clearly distinguish between fee rewards and staking rewards, I suggest naming:
`CurrentEraTransactionRewards -> CurrentEraFeeRewards`
and
`split_rewards_evenly_to_all -> split_fee_rewards_evenly_to_all`

Reply:
Good suggestion, will update soon